explainaboard/config.py

```diff
@@ -1,13 +1,14 @@
 from dataclasses import dataclass
+from typing import Optional
 
 
 @dataclass
 class BuilderConfig:
```

Comment:
I think this is unused code, it can just be deleted.

Reply:
Removed: 3009a3e